freqtrade/edge/edge_positioning.py

```diff
@@ -159,7 +159,7 @@ def calculate(self, pairs: List[str]) -> bool:
         logger.info(f'Measuring data from {min_date.strftime(DATETIME_PRINT_FORMAT)} '
                     f'up to {max_date.strftime(DATETIME_PRINT_FORMAT)} '
                     f'({(max_date - min_date).days} days)..')
-        #  TODO-lev: Should edge support shorts? needs to be investigated further...
+        #  * Should edge support shorts? needs to be investigated further (add enter_short exit_short)
```

Comment:
We won't be implementing edge immediately, so I thought this was fine to remove as a todo

Reply:
yes - at least as "TODO-lev". I've readded it as regular TODO - otherwise it's a regular comment and won't be found if someone is looking for "tasks".